### ISSUE TITLE
Developers/Newcomers_Guide.rst: Fix the broken link

### ISSUE THREAD

**Issue by aptrishu** [coala/documentation#172]
In **Step 3. Picking Up an Issue**

> Before starting to write your first commit, check out this link: :doc:`Writing good commits <Writing_Good_Commits>`.

Link mentioned in this line is broken.  
It should be referenced to `http://coala.io/commit`

**Comment by Adrianzatreanu** [comment]
since this doesnt have a PR yet, im closing this in favor of opening an issue that holds all broken links since this is quite urgent. https://github.com/coala/documentation/issues/173
